commands/lsp_metals_focus.py

```diff
@@ -1,14 +1,12 @@
-from LSP.plugin.core.registry import LspTextCommand
-from LSP.plugin.core.types import Optional, Dict, Any
+from . lsp_metals_text_command import LspMetalsTextCommand
 from LSP.plugin.core.protocol import Notification
+from LSP.plugin.core.types import Optional, Dict, Any
 
 import sublime
 import sublime_plugin
 import functools
 
-class LspMetalsFocusViewCommand(LspTextCommand):
-
-    session_name = "metals"
+class LspMetalsFocusViewCommand(LspMetalsTextCommand):
 
     def run(self, edit: sublime.Edit, view_id: int, event: Optional[Dict[str, Any]] = None) -> None:
```

Comment:
Pointless to send a `view_id` to a Text Command since it's already handled by that specific view.